README.md

```diff
@@ -1,25 +1,82 @@
-# `corppa`  PPA full-text corpus utilities
+# corppa
+
+This repository is research software developed as part of the [Ends of Prosody](https://cdh.princeton.edu/projects/the-ends-of-prosody/), which is associated with the [Princeton Prosody Archive](https://prosody.princeton.edu/) (PPA). This software is particularly focused on research and work related to PPA full-text and page image corpora.
+
+> [!WARNING]
+> This code is primarily for internal team use. Some portions of it may eventually be useful for participants of the [Ends of Prosody conference](https://cdh.princeton.edu/events/the-ends-of-prosody/) or be adapted or used elsewhere.
+
+## Basic Usage
+
+### Installation
+
+Use pip to install as a python package directly from GitHub.  Use a branch or tag name, e.g. `@develop` or `@0.1` if you need to install a specific version.
+
+```sh
+pip install git+https://github.com/Princeton-CDH/ppa-nlp.git#egg=corppa
+```
+or
+```sh
+pip install git+https://github.com/Princeton-CDH/ppa-nlp.git@v0.1#egg=corppa
+```
+
+### Scripts
+
+Installing `corppa` currently provides access to two command line scripts, for filtering a PPA page-level corpus or for generating OCR text for images using Google Vision API. These can be run as `corppa-filter-corpus` and `corppa-ocr` respectively.
+
+#### Filtering PPA page-text corpus
+
+The PPA page-level text corpus is shared as a json lines (`.jsonl`) file, which may or may not be compressed (e.g., `.jsonl.gz`). It's often useful to filter the full corpus to a subset of pages for a specific task, e.g. to analyze content from specific volumes or select particular pages for annotation.
+
+To create a subset corpus with _all pages_ for a set of specific volumes, create a text file with a list of **PPA work identifiers**, one id per line, and then run the filter script with the input file, desired output file, and path to id file.
+
+```sh
+corppa-filter-corpus ppa_pages.jsonl my_subset.jsonl --idfile my_ids.txt
```

Comment:
Do you think we could shorten to just corppa-filter ? It's such a long script name